tools/add_triage_signature.py

```diff
@@ -48,33 +53,60 @@
 """
 
 
+def config_logger(verbose: bool):
+    global logger
+
+    handler = colorlog.StreamHandler()
+    handler.setFormatter(colorlog.ColoredFormatter('%(log_color)s%(levelname)-10s %(message)s'))
+
+    logger.setLevel(logging.INFO)
+    logger.addHandler(handler)
+
+    if verbose:
+        logger.setLevel(logging.DEBUG)
+
+
 def format_description(failure_data):
     return JIRA_DESCRIPTION.format(**failure_data)
 
+
 def days_ago(datestr):
     try:
         return (datetime.now() - dateutil.parser.isoparse(datestr)).days
     except Exception:
         logger.debug("Cannot parse date: %s", datestr)
         return 9999
 
+
 @functools.lru_cache(maxsize=1000)
 def get_metadata_json(cluster_url):
     res = requests.get("{}/metdata.json".format(cluster_url))
     res.raise_for_status()
     return res.json()
 
+
 @functools.lru_cache(maxsize=1000)
 def get_events_json(cluster_url, cluster_id):
     res = requests.get(f"{cluster_url}/cluster_{cluster_id}_events.json")
     res.raise_for_status()
     return res.json()
 
+
+@functools.lru_cache(maxsize=1000)
+def get_cluster_logs(cluster_url: str, cluster_id: str, output_file: str):
+    res = requests.get(f"{cluster_url}/cluster_{cluster_id}_logs.tar")
+    res.raise_for_status()
+
+    with open(output_file, "wb") as _file:
```

Comment:
`_` is typically used to denote private or unused. I think it's safe/common to just use `f` in this case